### ISSUE TITLE
highlight changed parameters

### ISSUE THREAD

**Issue by mlindauer** [sfalkner/SpySMAC#4]
it is not obvious what changed in the final incumbent in comparison to the default configuration.
we should highlight and sort the parameters based on whether their value changed or not.

**Comment by sfalkner** [comment]
I agree, we should improve the formatting of the final configuration. Maybe a table divided into changed and unchanged. If the fANOVA runs through, one could even sort the table by 'importance'.
